app/src/main/java/io/intrepid/dictionary/DictionaryPresenter.kt

```diff
@@ -0,0 +1,46 @@
+package io.intrepid.dictionary
+
+import retrofit2.Call
+import retrofit2.Callback
+import retrofit2.Response
+import timber.log.Timber
+
+class DictionaryPresenterImpl(
```

Comment:
I just noticed that file name is DictionaryPresenter.kt, whereas class is DictionaryPresenterImpl; is that a Kotlin thing? might you define multiple impl classes in the same file here?

Reply:
Oh oops, no that was a missed rename on my part, I'll change that